RxCocoa/iOS/NSTextStorage+Rx.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  NSTextStorage+Rx.swift
+//  Rx
+//
+//  Created by Segii Shulga on 12/30/15.
+//  Copyright © 2015 Krunoslav Zaher. All rights reserved.
+//
+
+#if os(iOS) || os(tvOS)
+    
+    import Foundation
+    import UIKit
+#if !RX_NO_MODULE
+    import RxSwift
+#endif
+
+extension NSTextStorage {
+    
+    public var rx_delegate:DelegateProxy {
+        return proxyForObject(RxTextStorageDelegateProxy.self, self)
+    }
+    
+    public var rx_string:Observable<String> {
```

Comment:
The convention is for delegate proxies to expose a method that is named the same like corresponding delegate method.

You can find samples in other delegate proxies.